autoPyTorch/pipeline/tabular_classification.py

```diff
@@ -243,7 +243,7 @@ def _get_pipeline_steps(self, dataset_properties: Optional[Dict[str, Any]],
             ("preprocessing", EarlyPreprocessing()),
             ("network_backbone", NetworkBackboneChoice(default_dataset_properties)),
             ("network_head", NetworkHeadChoice(default_dataset_properties)),
-            ("network", NetworkComponent(default_dataset_properties)),
```

Comment:
Do you think we can try to keep the same interface to other components?

Reply:
I thought this might be cleaner, since the NetworkComponent similar to the EarlyPreprocessing just aggregates parts of the pipeline (backbone and head for the NetworkComponent, all the preprocessing transformers for EarlyPreprocessing). They dont require dataset properties because of that, and because it was already removed at the EarlyPreprocessing I also removed it here. What do you think?